docs/core/tools/dotnet-migrate.md

```diff
@@ -1,23 +1,16 @@
 ---
 title: dotnet migrate command
 description: The dotnet migrate command migrates a project and all of its dependencies.
-ms.date: 06/26/2019
+ms.date: 08/08/2019
```

Comment:
I'll stop mentioning, but all ms.date fields should be updated before merging.

Reply:
LOL thanks! I'll fix those.

Reply:
Oh but I did change the ms.date. I"ll double check for Preview 9/RTM if anything changed again and do a pass across all CLI ref. This date is good as-is.